test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -318,6 +318,7 @@ class GrpcJsonTranscoderFilterTest : public testing::Test, public GrpcJsonTransc
     const std::string json_string = "{\"proto_descriptor\": \"" + bookstoreDescriptorPath() +
                                     "\",\"services\": [\"bookstore.Bookstore\"]}";
     envoy::config::filter::http::transcoder::v2::GrpcJsonTranscoder proto_config;
+    proto_config.set_convert_grpc_status(true);
```

Comment:
why are you testing this config here? there's another set of test (`GrpcJsonTranscoderFilterConvertGrpcStatusTest`) to test this.